clients/src/main/java/org/apache/kafka/common/protocol/types/Type.java

```diff
@@ -320,7 +320,7 @@ public Long validate(Object item) {
             if (item instanceof Long)
                 return (Long) item;
             else
-                throw new SchemaException(item + " is not a Long.");
+                throw new SchemaException(item + " is not an a Long (encoding an unsigned integer).");
```

Comment:
There is a typo here, "an a".

Reply:
Also, I think it reads a bit weird. Not clear that "encoding an unsigned integer" in brackets means when reading the message.

Reply:
> There is a typo here, "an a".

My bad :(

> Also, I think it reads a bit weird. Not clear that "encoding an unsigned integer" in brackets means when reading the message.

I will update it in #10248 @ijuma thanks for your reviews